data/korath outfits.txt

```diff
@@ -310,3 +310,12 @@ outfit "Steering (Stellar Class)"
 	"turning energy" 8.2
 	"turning heat" 26.7
 	description "After millennia of war, the Korath have developed some truly enormous warships. These steering systems give even their largest warships a reasonable amount of agility in combat."
+	
+outfit "Korath Repeater Rifle"
+	category "Hand to Hand"
+	cost 220000
+	thumbnail "outfit/korath rifle"
+	"capture attack" 3.1
+	"capture defense" 5.4
+	"unplunderable" 1
+	description "The Korath Repeater Rifle, while commonplace among the Korath, is considered highly illegal by everyone but the lawless who see them as trophies of sorts and will pay nicely for it."
```

Comment:
Since it's no longer illegal, this is no longer accurate.

Reply:
look i will make it illegal once i can choose who its illegal for but till then everyone loves it